### PR TITLE
Virtual routing fix

### LINKED ISSUE #211: 🐛 Cannot resolve dependency when I use <a> tag with virtual path
🐛 bug report

### 🎛 Configuration (.babelrc, package.json, cli command)

package.json
```
{
  "name": "parcel-test",
  "version": "0.1.0",
  "devDependencies": {
    "parcel-bundler": "git://github.com/parcel-bundler/parcel.git"
  }
}
```
index.html
```
<html>
  <body>
    <a href="/b">CLICK ME</a>
  </body>
</html>
```
And run command:
```
parcel build src/index.html
```

### 🤔 Expected Behavior

No error

### 😯 Current Behavior

Thrown an error with message `Cannot resolve dependency './..\..\..\..\b'`

### 🌍 Your Environment

| Software         | Version(s)
| ---------------- | ----------
| Parcel           |  master branch
| Node             |  8.9.3
| npm/Yarn         |  Yarn 1.2.1
| Operating System |  Windows 10 x64

### PR DESCRIPTION
**Pull request by DeMoorJasper**
potential fix for closes #211

### CODE REVIEW

**Comment by albinotonnina** [comment]
You are on fire @DeMoorJasper! 🙌👏

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: src/assets/HTMLAsset.js [hunk 1]
```diff
@@ -45,6 +45,10 @@ class HTMLAsset extends Asset {
       if (node.attrs) {
         for (let attr in node.attrs) {
           let elements = ATTRS[attr];
+          // link is not a file
+          if (node.tag === 'a' && node.attrs[attr].lastIndexOf('.') < 1) {
```
Comment (devongovett) [inline src/assets/HTMLAsset.js]: Maybe we should just ignore all absolute paths within HTML instead? so just check if the first character is `/`?

Comment (DeMoorJasper) [reply src/assets/HTMLAsset.js]: Wouldn't this also break any absolute path require, (that has already been reported as a bug)?

Comment (devongovett) [reply src/assets/HTMLAsset.js]: Not sure why you'd want to require an absolute path so this might be ok.

**Comment by DeMoorJasper** [comment]
Conflicts have been fixed @brandon93s

**Pull request merged**